models/mail.go

```diff
@@ -152,10 +152,13 @@ func composeTplData(subject, body, link string) map[string]interface{} {
 
 func composeIssueCommentMessage(issue *Issue, doer *User, content string, comment *Comment, tplName base.TplName, tos []string, info string) *mailer.Message {
 	subject := issue.mailSubject()
-	issue.LoadRepo()
+	err := issue.LoadRepo()
+	if err != nil {
+		log.Error("LoadRepo: %v", err)
+	}
 	body := string(markup.RenderByType(markdown.MarkupName, []byte(content), issue.Repo.HTMLURL(), issue.Repo.ComposeMetas()))
 
-	data := make(map[string]interface{}, 10)
+	var data = make(map[string]interface{}, 10)
```

Comment:
The created slice is never used.
```suggestion
	var data map[string]interface{}
```

Reply:
It is, in line 171 the map is passed to `templates.ExecuteTemplate`.

Reply:
The value setted on line 161 is overwritten on line 163 or 165.

Reply:
But your suggestion would not pre-allocate a map with 10 items, only make it available.

Reply:
This seems unnecessary because it will not be used. I think that's what raised this issue in the first place IMHO.